pandas/core/indexes/base.py

```diff
@@ -3396,11 +3396,8 @@ def map(self, mapper, na_action=None):
 
     def isin(self, values, level=None):
         """
-        Return a boolean array where the index values are in `values`.
```

Comment:
Can you revert the changes here? Let me know if you need help with that.

Reply:
@TomAugspurger - I had removed it as you had requested to only have the deprecation code in the branch / PR. So just to clarify you do want this code back in this branch but you don't want that large pr file back in the branch?